patchwork-tags/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,32 @@
+{
+  "schemaVersion": 1,
+  "id": "patchwork-tags",
+  "name": "Patchwork Tags",
+  "version": "${version}",
+  "license": "LGPL-2.1-only",
+  "icon": "assets/patchwork-tags/icon.png",
+  "contact": {
+    "issues": "https://github.com/PatchworkMC/patchwork-api/issues",
+    "sources": "https://github.com/PatchworkMC/patchwork-api"
+  },
+  "authors": [
+    "PatchworkMC"
+  ],
+  "accessWidener": "patchwork-tags.accesswidener",
+  "depends": {
+    "fabricloader": ">=0.8.4",
+    "patchwork-fml": "*"
+  },
+  "description": "Implementation of the Forge Tags API.",
+  "entrypoints": {
```

Comment:
Pretty sure this can be left out if you don't have any entrypoints.

Reply:
Confirmed, we can have a Fabric mod without any entrypoint

Reply:
I also removed the unnecessary fml dependency